tailwind.config.js

```diff
@@ -3,6 +3,35 @@ module.exports = {
   purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
   darkMode: false,
   theme: {
+    fontFamily: {
+      display: ['Lato'],
+      body: ['Noto sans'],
+    },
+    fontSize: {
```

Comment:
can we not add these from alpha, it is really messy. Just add what we need when we need it?

Reply:
I am using the font family in the footer but I will remove what we don't need.